app/controllers/api/v2/provisioning_templates_controller.rb

```diff
@@ -132,9 +132,12 @@ def export
       end
 
       api :GET, '/register', N_('Render Global Registration template')
+      param :organization_id, :number, required: true, desc: N_("ID of the Organization to register the host in.")
+      param :location_id, :number, desc: N_("ID of the Location to register the host in.")
```

Comment:
both defaults to `Organization.current`/`Location.current`, why only organization is required?